v1/endpoints.md

```diff
@@ -856,6 +860,17 @@ Response codes:
 - `400 Bad request`
 - `403 Unauthorized`
 
+### Free-text search tips and tricks
+
+We recommend taking the following precautions when performing free-text searches 
+
+* explicitly override the `documentVersions` `attribute`s to be returned in the response as the `text` attribute may unnecessarily increase the payload size significantly for documents that contain a lot of text
+* consider making use of text highlight snippets (`highlights` returned for each resource) instead of retrieving the unabbreviated text attribute (`text`) that may be significantly larger  
```

Comment:
Trivial nitpick; missing a period at the end of this sentence? Ref the next line with an upper case that seems to be part of the same bullet?

Reply:
Ah, good catch. I'll make not of it to fix it together with a future PR.